parsers/CR.py

```diff
@@ -7,115 +7,26 @@
-    "Birris3": "hydro",
-    "Boca de Pozo": "hydro",
-    "CNFL": "hydro",
-    "Cachí": "hydro",
-    "Campos Azules": "wind",
-    "Canalete": "hydro",
-    "Cariblanco": "hydro",
-    "Carrillos": "hydro",
-    "Caño Grande": "hydro",
-    "Caño Grande III": "hydro",
-    "Chiripa": "wind",
-    "Chocosuelas": "hydro",
-    "Chucás": "hydro",
-    "Cote": "hydro",
-    "Cubujuquí": "hydro",
-    "Daniel Gutiérrez": "hydro",
-    "Dengo": "hydro",
-    "Don Pedro": "hydro",
-    "Doña Julia": "hydro",
-    "Echandi": "hydro",
-    "Electriona": "hydro",
-    "El Encanto": "hydro",
-    "El Angel": "hydro",
-    "El Angel Ampliación": "hydro",
-    "El Embalse": "hydro",
-    "El General": "hydro",
-    "El Viejo": "biomass",
-    "Garabito": "oil",
-    "Garita": "hydro",
-    "Guápiles": "oil",
-    "Hidrozarcas": "hydro",
-    "Jorge Manuel Dengo": "hydro",
-    "La Esperanza (CoopeL)": "hydro",
-    "La Joya": "hydro",
-    "Las Pailas II": "geothermal",
-    "Los Negros": "hydro",
-    "Los Negros II": "hydro",
-    "Los Santos": "wind",
-    "MOVASA": "wind",
-    "Matamoros": "hydro",
-    "Miravalles I": "geothermal",
-    "Miravalles II": "geothermal",
-    "Miravalles III": "geothermal",
-    "Miravalles V": "geothermal",
-    "Moín I": "oil",
-    "Moín II": "oil",
-    "Moín III": "oil",
-    "Orosí": "wind",
-    "Orotina": "unknown",
-    "Otros": "unknown",
-    "PE Cacao": "wind",
-    "PE Mogote": "wind",
-    "PE Río Naranjo": "hydro",
-    "PEG": "wind",
-    "Pailas": "geothermal",
-    "Parque Solar Juanilama": "solar",
-    "Parque Solar Miravalles": "solar",
-    "Peñas Blancas": "hydro",
-    "Pirrís": "hydro",
-    "Plantas Eólicas": "wind",
-    "Platanar": "hydro",
-    "Pocosol": "hydro",
-    "Poás I y II": "hydro",
-    "Reventazón": "hydro",
-    "Río Lajas": "hydro",
-    "Río Macho": "hydro",
-    "Río Segundo": "hydro",
-    "San Antonio": "oil",
-    "San Lorenzo (C)": "hydro",
-    "Sandillal": "hydro",
-    "Suerkata": "hydro",
-    "Taboga": "biomass",
-    "Tacares": "hydro",
-    "Tejona": "wind",
-    "Tilawind": "wind",
-    "Torito": "hydro",
-    "Toro I": "hydro",
-    "Toro II": "hydro",
-    "Toro III": "hydro",
-    "Tuis (JASEC)": "hydro",
-    "Valle Central": "wind",
-    "Vara Blanca": "hydro",
-    "Ventanas": "hydro",
-    "Ventanas-Garita": "hydro",
-    "Vientos de La Perla": "wind",
-    "Vientos de Miramar": "wind",
-    "Vientos del Este": "wind",
-    "Volcán": "hydro",
+EXCHANGE_URL = (
+    "https://mapa.enteoperador.org/WebServiceScadaEORRest/webresources/generic"
+)
+
+SPANISH_TO_ENGLISH = {
+    "Bagazo": "biomass",
+    "Eólica": "wind",
+    "Geotérmica": "geothermal",
+    "Hidroeléctrica": "hydro",
+    "Solar": "solar",
+    "Térmica": "oil",
 }
 
-CHARACTERISTIC_NAME = "Angostura"
+EXCHANGE_JSON_MAPPING = {
+    "CR->NI": "5SISTEMA.LT230.INTER_NET_CR.CMW.MW",
+    "CR->PA": "6SISTEMA.LT230.INTER_NET_PAN.CMW.MW",
+}
 
 
 def empty_record(zone_key: str):
```

Comment:
We should never return a manually modified datapoint, especially not a 0 filled one as it will break out ability to estimate the zone in the future.

Reply:
What do you mean by "manually modified"? All "empty records" are going to be filled later on from line 118 through 123. I first just create the empty dictionary and populate it afterwards.